src/reformat.c

```diff
@@ -180,7 +190,7 @@ static int avifReformatStateUVToUNorm(avifReformatState * state, float v)
 
 avifResult avifImageRGBToYUV(avifImage * image, const avifRGBImage * rgb)
 {
-    if (!rgb->pixels) {
+    if (!rgb->pixels || rgb->format == AVIF_RGB_FORMAT_RGB_565) {
```

Comment:
Should this return `AVIF_RESULT_NOT_IMPLEMENTED`?

Reply:
it is very unlikely that we will ever support this conversion since there are no image formats where the source is rgb565. so i decided not to return `AVIF_RESULT_NOT_IMPLEMENTED` here.

This is also documented int he AVIF_RGB_FORMAT_RGB_565 enum definition.